orchestrator-salt/src/main/resources/salt-common/salt/monitoring/scripts/logging-agent.sh.j2

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+if [[ -f "/var/run/cdp-logging-agent/cdp-logging-agent.pid" ]]; then
+  platform="{{ salt['pillar.get']('fluent:platform') }}"
+  logging_types=""
+{%- if salt['pillar.get']('fluent:cloudStorageLoggingEnabled') %}
+  if [[ "$platform" == "AWS" ]]; then
+    logging_types="s3"
+  elif [[ "$platform" == "AZURE" ]]; then
+    logging_types="abfs"
+  elif [[ "$platform" == "GCP" ]]; then
+    logging_types="gcs"
+  fi
+  use_cloud_storage="true"
+{%- else %}
+  use_cloud_storage="false"
+{%- endif %}
+{%- if salt['pillar.get']('fluent:dbusMeteringEnabled') %}
+  logging_types="$logging_types metering_databus"
+{%- endif %}
+{%- if salt['pillar.get']('fluent:dbusClusterLogsCollection') %}
+  logging_types="$logging_types databus_service_logs"
+{%- endif %}
+  if [[ "$logging_types" != "" ]]; then
+    logging_types=$(echo "$logging_types" | sed 's/[ ]*$//')
+    logging_agent_process_details=$(ps aux | grep fluentd | grep -v grep | tr -s '[:blank:]' ',')
+    logging_agent_process_arr=($(echo "$logging_agent_process_details"))
+    declare -p logging_agent_process_arr > /dev/null
+    IFS=' ' read -a logging_types_arr <<< "$logging_types"
+    i=0
+    for logging_agent_process in ${logging_agent_process_arr[@]}
+    do
+      pid=$(echo "$logging_agent_process" | cut -d ',' -f2)
+      rss=$(echo "$logging_agent_process" | cut -d ',' -f6)
+      rss_bytes=$(($rss*1024))
+      fds_num=$(ls -la "/proc/$pid/fd" | wc -l)
+      if [[ "$logging_agent_process" == *"under-supervisor"* ]]; then
```

Comment:
what is `under-supervisor`? why is it necassary here? please add a comment or commit message